src/plugins/intel_cpu/src/nodes/kernels/x64/random_uniform.cpp

```diff
@@ -0,0 +1,642 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "random_uniform.hpp"
+
+using namespace dnnl::impl::cpu;
+
+namespace ov {
+namespace intel_cpu {
+namespace kernel {
+
+#define GET_OFF(field) offsetof(RandomUniformCallArgs, field)
+
+template <x64::cpu_isa_t isa>
+RandomUniform<isa>::RandomUniform(const RandomUniformCompileParams& jcp) :
+        JitKernel(jit_name(), jcp, isa) {
+}
+
+template <x64::cpu_isa_t isa>
+void RandomUniform<isa>::generate() {
+    this->preamble();
+    registersPool = RegistersPool::create(isa, {rax, rcx, rsp, rdi, k0});
+
+    r64_dst = getReg64();
+    r64_work_amount = getReg64();
+
+    mov(r64_work_amount, ptr[regParams + GET_OFF(work_amount)]);
+    mov(r64_dst,  ptr[regParams + GET_OFF(dst_ptr)]);
+
+    initVectors();
+    process();
+
+    registersPool.reset();
+    this->postamble();
+}
+
+template <>
+void RandomUniform<x64::avx512_core>::initVectors() {
+    const auto r64_aux = getReg64();
+    const auto r32_aux = Xbyak::Reg32(r64_aux.getIdx());
+    const auto r16_aux = Xbyak::Reg16(r64_aux.getIdx());
+
+    v_max_mul_n_64 = getVmm();
+    v_max_mul_c_64 = getVmm();
+    v_add_low_k    = getVmm();
+    v_add_up_k     = getVmm();
+    v_convert_0    = getVmm();
+    v_convert_1    = getVmm();
+    v_n_inc        = getVmm();
+    v_range        = getVmm();
+    v_min          = getVmm();
+    v_key_64       = getVmm();
+    v_counter_64   = getVmm();
+    v_n_64         = getVmm();
+    v_res_perm     = getVmm();
+
+    // Initialize constants.
+    mov(r64_aux, 0xd2511f53);
```

Comment:
Could you please leave comments what is the constants?

Reply:
Moreover, I see you use these constants in different places. Maybe we can create static class fields with clear names with that are equal to these values?